invenio_rdm_records/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from flask_security.confirmable import confirm_user
+from flask_security.utils import hash_password
+from invenio_accounts.proxies import current_datastore
+from invenio_db import db
+from invenio_users_resources.services.users.tasks import reindex_user
+
+
+def get_or_create_user(email):
+    """Get or create a user."""
+    user = current_datastore.get_user(email)
+    if not user:
+        with db.session.begin_nested():
+            user = current_datastore.create_user(
+                email=email,
+                password=hash_password("123456"),
```

Comment:
Password is in plain text, one can login in any instance with this user (unless the user is removed)

Reply:
Yes, this is the "default" one created if you run the users fixtures and no user was provided, this can be overriden by creating the same user in the users.yaml.

Reply:
I would suggest to change it somehow to not have the password in plain text (perhaps using a secret in an environment variable). I was able to login to a prod instance with this user.

Otherwise, it should be clear that this user must be deleted in production or it should be disabled by default.

Reply:
IMHO this is never meant to be run in production, these wil create demo data and demo users, therefore I don't see the need to change anything but I am happy to discuss this quickly IRL or in the stand-up to see other people's opinion on this

Reply:
I agree, we shouldn't expose any passwords in plain text, since we have demo instances available publicly. Also to take a good habit